data/json/construction.json

```diff
@@ -4226,10 +4226,34 @@
     "required_skills": [ [ "fabrication", 3 ], [ "mechanics", 2 ], [ "tailor", 1 ] ],
     "time": "3 h",
     "qualities": [ [ { "id": "CUT", "level": 1 } ], [ { "id": "SAW_W", "level": 1 } ], [ { "id": "HAMMER", "level": 1 } ] ],
-    "components": [ [ [ "2x4", 20 ] ], [ [ "string_36", 15 ] ], [ [ "nail", 25 ] ] ],
+    "components": [ [ [ "2x4", 20 ] ], [ [ "string_36", 4 ] ], [ [ "nail", 25 ] ] ],
     "pre_special": "check_empty",
     "post_terrain": "f_spinwheel"
   },
+  {
+    "type": "construction",
+    "id": "constr_spin_wheel_cordage",
+    "group": "build_spinwheel",
+    "category": "FURN",
+    "required_skills": [ [ "fabrication", 3 ], [ "mechanics", 2 ], [ "tailor", 1 ] ],
+    "time": "3 h",
+    "qualities": [ [ { "id": "CUT", "level": 1 } ], [ { "id": "SAW_W", "level": 1 } ], [ { "id": "HAMMER", "level": 1 } ] ],
+    "components": [ [ [ "2x4", 20 ] ], [ [ "cordage_36", 4 ] ], [ [ "nail", 25 ] ] ],
```

Comment:
So this needs 20 2x4's and 25 nails?

Reply:
That....does seem like a lot.  Those were the previous values, but I'm happy to update them to something more realistic.  Then again if you look at images of a Spinning Jenny (Which is what I'm told this is supposed to be) maybe it's right.  

Honestly though I'm not anywhere near an expert here.  If you think it should be different, just name some numbers and I'll make the changes.

Reply:
Isn't the Spinning Wheel supposed to be the tool that's been used for local use in farms for hundreds of years, rather than the industrial revolution automated version of it?

Reply:
Ok let’s assume the two by fours are right then it will need 80 nails four in each two by four is a much better average than 1.2 nails per two by four.

Reply:
80 it is.  Thanks!

Reply:
using these two designs as a reference: http://granitemountainwoodcraft.com/2015/03/07/making-a-spinning-wheel/

https://www.craftsmanspace.com/free-projects/spinning-wheel-plan.html

our planks are ~51” long

Not going to do the math on the size for final dimensions, and the design will be closer to the first design, which seemed to be more popular historically. looking at the above plans, the outer wheel itself would take 6 planks to construct. Cut in half lengthwise, you’d need a set of 3 for the large boards and another set of 3 for the trapezoids to cover the extra curve. You’d need to sink the nails pretty deep, but 4 per board on average should build the wheel in a way that should hold up to the rotational force.

Current count:

6 planks, 24 nails

you’ll need another set of 3 planks to build spokes for stability, and a partial chunk for the hub. We’ll use the rest of the partial chunk for the base. You’ll need a handful of nails for the hub, we’ll handle the base next

10 planks, 30 nails

Drilling or even a chisel would be super helpful here, but you could use nails as a primitive drill. 

Now we need the axel and arms to hold the wheel. You’ll need more than half a plank per arm to give the wheel clearance, so let’s call it two planks and make the axel out of one of the scraps there. Two nails to attach the axel, two to four nails to attach the arms to the base

12 planks, 36 nails. 

Four legs to attach to the base. You could get by with three, but eh. Each is half a plank. Two nails per leg to help secure it. 

16 planks, 44 nails. 

The flyer looks pretty damn complex with lots of room for failure, even with tools and experience. But ideally, the flyer and the holder shouldn’t be more than a plank’s worth of wood. You’ll need 1 nail for the flyer axel and then 4 or so to mount the holder to the frame. The hooks on the flyer could be made of bent nails. 

17 planks, ~60 nails. That’s my best guess on the ratio. 

Additionally, you’ll want filing 1. You’re going to need some way to sand it smooth if you don’t want to destroy the fiber as it’s spinning. Also the bobbin will need smoothed.